ArmaForces.Boderator/ArmaForces.Boderator.BotService/Controllers/DLCController.cs

```diff
@@ -0,0 +1,162 @@
+using System;
+using System.Linq;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json.Linq;
+using System.Collections.Generic;
+using System.IO;
+using ArmaForces.Boderator.BotService.DataClasses;
+
+namespace ArmaForces.Boderator.BotService.Controllers
+{
+    [Route("users/")]
+    [ApiController]
+    [Produces("application/json")]
+    public class DLCController : ControllerBase
+    {
+        private readonly Users _users;
+
+        public DLCController(Users users)
+        {
+            _users = users;
+        }
+
+        //Adds DLC to user
+        [HttpPut("{userID}/dlcadd/{dlcName}")]
+        public void AddDlcToUser(ulong userID, SingleUser.DLC dlcName)
+        {
+            if (!(_users.UsersList.Any(x => x.UserID == userID)))
+            {
+                Response.StatusCode = 404;
+                Response.WriteAsync($"User not found");
+                return;
+            }    
+
+            if (_users.UsersList.Where(x => x.UserID == userID).Any(x => x.DLCList.Contains(dlcName)))
+            {
+                Response.WriteAsync($"DLC already added");
+                return;
+            }
+
+            _users.UsersList.Find(x => x.UserID == userID).DLCList.Add(dlcName);
+
+            Response.WriteAsync($"DLC added");
+            return;
+        }
+
+        //List user with given ID. "0" lists all users
+        [HttpGet("{userID}")]
+        public void UserOnServer(ulong userID)
```

Comment:
No. You should have separate method for no input instead of special handling for `0` value. And method should be named `GetUser`. Then method for all users would just be named `GetUsers` or `GetAllUsers`.